packages/instantsearch.js/src/lib/server.ts

```diff
@@ -37,17 +57,27 @@ export function waitForResults(search: InstantSearch): Promise<void> {
 /**
  * Walks the InstantSearch root index to construct the initial results.
  */
-export function getInitialResults(rootIndex: IndexWidget): InitialResults {
+export function getInitialResults(
+  rootIndex: IndexWidget,
+  /**
+   * Search parameters sent to the search client, usually
+   * returned by `waitForResults()`.
+   */
+  requestParamsList?: RequestParams[]
+): InitialResults {
   const initialResults: InitialResults = {};
 
   walkIndex(rootIndex, (widget) => {
     const searchResults = widget.getResults();
     if (searchResults) {
-      initialResults[widget.getIndexId()] = {
```

Comment:
I'm fine with keeping the indexId variable, but just making sure that it's not doing anything now

Reply:
I can revert that you're right.